dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/AssertionsTest.java

```diff
@@ -49,58 +51,55 @@
     @Test
     void testAssertHasOnlyErrorsFailsIfReportHasNoErrors()
     {
-
-        ValidationReport report = new ValidationReport();
+        ValidationReport report = ValidationReport.emptyReport();
 
         assertThrows( AssertionError.class, () -> assertHasOnlyErrors( report, TrackerErrorCode.E1000 ) );
-        // assertHasOnlyErrors(report, TrackerErrorCode.E1000);
     }
 
     @Test
     void testAssertHasOnlyErrorsFailsIfReportHasMoreErrors()
     {
 
-        ValidationReport report = new ValidationReport();
-        report.addError( TrackerErrorReport.builder().errorCode( TrackerErrorCode.E1000 ).build() );
-        report.addError( TrackerErrorReport.builder().errorCode( TrackerErrorCode.E1019 ).build() );
-        report.addError( TrackerErrorReport.builder().errorCode( TrackerErrorCode.E1041 ).build() );
+        ValidationReport report = new ValidationReport(
+            List.of( Error.builder().errorCode( TrackerErrorCode.E1000.name() ).build(),
```

Comment:
At some point we should remove the Error.builder as all fields are required. We can add a factory if that makes it nicer to read than `new Error()`